src/Compilers/Core/Portable/CodeGen/LocalScopeManager.cs

```diff
@@ -739,8 +739,6 @@ public override void CloseScope(ILBuilder builder)
 
             internal override void GetExceptionHandlerRegions(ArrayBuilder<Cci.ExceptionHandlerRegion> regions)
             {
-                Debug.Assert(_handlers.Count > 1);
-
                 ExceptionHandlerScope tryScope = null;
```

Comment:
Your removing the assertion here implies that `_handlers.Count` can be zero. Is that the case?